### DISCUSSION TITLE
Set Labels for Dataset

### DISCUSSION THREAD

**Discussion by chr-frick** [dbt-labs/dbt-core#6225] [Q&A]
I am trying to set a Label for my Dataset in Bigquery, preferably just once on dataset creation. 

ALTER SCHEMA mydataset
SET OPTIONS (  labels = [('sensitivity', 'high')]);

How can I manipulate the creation of a dataset?

**Comment by jtcohen6** [comment]
Related discussion: https://github.com/dbt-labs/dbt-core/discussions/5781
